esmerald/datastructures/encoders.py

```diff
@@ -19,7 +19,7 @@
     UJSONResponse = None  # type: ignore
 
 
-class OrJSON(ResponseContainer[ORJSONResponse]):
+class ORJSON(ResponseContainer[ORJSONResponse]):
```

Comment:
Ok I will let this one pass but this is irrelevant as it's a style that is a preference. Just for reference

Reply:
I understood and that is why I'm letting pass here 👍🏼